tests/components/flo/test_binary_sensor.py

```diff
@@ -0,0 +1,30 @@
+"""Test Flo by Moen binary sensor entities."""
+from homeassistant.components.flo.const import DOMAIN as FLO_DOMAIN
+from homeassistant.const import (
+    ATTR_FRIENDLY_NAME,
+    CONF_PASSWORD,
+    CONF_USERNAME,
+    STATE_ON,
+)
+from homeassistant.setup import async_setup_component
+
+from .common import TEST_PASSWORD, TEST_USER_ID
+
+
+async def test_binary_sensors(hass, config_entry, aioclient_mock_fixture):
+    """Test Flo by Moen sensors."""
+    config_entry.add_to_hass(hass)
+    assert await async_setup_component(
+        hass, FLO_DOMAIN, {CONF_USERNAME: TEST_USER_ID, CONF_PASSWORD: TEST_PASSWORD}
+    )
+    await hass.async_block_till_done()
+
+    assert len(hass.data[FLO_DOMAIN]["devices"]) == 1
+
+    # we should have 6 entities for the device
+    state = hass.states.get("binary_sensor.pending_system_alerts")
+    assert state.state == STATE_ON
+    assert state.attributes.get("info") == 0
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)